drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/ScenarioSimulationEditorPresenter.java

```diff
@@ -504,18 +535,22 @@ public void populateRightDocks(String identifier) {
                         presenter.setCurrentPath(path);
                     });
                     break;
+                default:
```

Comment:
Could we add at least some log notification?

Reply:
@dupliaka Added an exception in that case: If switch goes to default case the passed value is invalid.